src/components/Layout/mdx/PageHeader.tsx

```diff
@@ -44,41 +45,80 @@ export const PageHeader: React.FC<PageHeaderProps> = ({ title, intro }) => {
     [activePage.languages, product],
   );
 
-  const handleCopyMarkdown = useCallback(async () => {
-    try {
-      const response = await fetch(`${location.pathname}.md`);
+  useEffect(() => {
+    const abortController = new AbortController();
```

Comment:
Lets us abort the fetch if dismounted, was playing with the tests a bit (and is a positive step)